docs/docs/examples.md

```diff
@@ -191,3 +191,35 @@ dissipation 1
 Running this example will take a bit longer than the others, as the program needs to evaluate the function a lot of times to find the correct value. At the end, it should converge on a value close to `g = 3.0`, and as we can see here, the fitted function matches the fake data quite well:
 
 ![](./figExFitting.png)
+
+## Example 10 - Celio's Method
+**Input file:** `examples/celio/celio.in`
+
+Finally, we look at how we can use Celio's method to approximate a large system with MuSpinSim. Here we give an example of $\text{V}_3\text{Si}$ as described in the paper
+
+> Pietro Bonfà, Jonathan Frassineti, John M. Wilkinson, Giacomo Prando, Muhammad Maikudi Isah, Chennan Wang, Tiziana Spina, Boby Joseph, Vesna F. Mitrović, Roberto De Renzi, Stephen J. Blundell, and Samuele Sanna
+> [Entanglement between Muon and $I > \frac{1}{2}$ Nuclear Spins as a Probe of Charge Environment](https://doi.org/10.1103/PhysRevLett.129.097205)
+> *Phys. Rev. Lett. **129**, 097205 – Published 26 August 2022*
+
+In brief since the Si atoms have 0 spin, we only consider the Vanadium atoms and then include the 4 nearest neighbours to the muon stopping site. We then consider the dipolar couplings of the Vanadium atoms with the muon and quadrupolar couplings of the Vanadium atoms. For the latter which requires EFG tensors, we use values obtained using GIPAW and average over simulations with muons initially polarised in the x, y and z directions via
+
+```plaintext
+polarization
+    1 0 0
+    0 1 0
+    0 0 1
+average_axes
+    polarization
+```
+
+We then request that MuSpinSim uses Celio's method using
+
+```plaintext
+celio
+    10 4
+```
+
+Where the first value is the trotter number $k = 10$, and the second value gets MuSpinSim to randomise the initial states and compute the average of 4 simulations for each initial polarisation direction.
```

Comment:
```suggestion
    8 4
```

Where the first value is the trotter number $k = 8$, and the second value gets MuSpinSim to randomise the initial states and compute the average of 4 simulations for each initial polarisation direction.
```
It's not a big deal, but in the `celio.in` file the trotter number is 8, not 10. I'm assuming the file is accurate but if it was actually 10 then obviously keep this as is and change the value in the file.